drivers/bus/acpi/compbatt/compbatt.c

```diff
@@ -20,31 +20,33 @@ ULONG CompBattDebug;
 
 NTSTATUS
 NTAPI
-CompBattOpenClose(IN PDEVICE_OBJECT DeviceObject,
-                  IN PIRP Irp)
+CompBattOpenClose(
+    _In_ PDEVICE_OBJECT DeviceObject,
+    _In_ PIRP Irp)
 {
     PAGED_CODE();
-    if (CompBattDebug & 0x100) DbgPrint("CompBatt: ENTERING OpenClose\n");
+    if (CompBattDebug & COMPBATT_DEBUG_FUNC_ENTER_EXIT) DbgPrint("CompBatt: ENTERING OpenClose\n");
```

Comment:
These look more like INFO vs. TRACE vs. WARN vs. ERR type of dprints (implemented with associated flags)